test/e2e/node/kubelet.go

```diff
@@ -458,187 +458,144 @@ var _ = SIGDescribe("kubelet", func() {
+		ginkgo.It("should return the kubelet logs ", func(ctx context.Context) {
 			ginkgo.By("Starting the command")
 			tk := e2ekubectl.NewTestKubeconfig(framework.TestContext.CertDir, framework.TestContext.Host, framework.TestContext.KubeConfig, framework.TestContext.KubeContext, framework.TestContext.KubectlPath, ns)
 
-			for nodeName := range nodeNames {
-				queryCommand := "\"/api/v1/nodes/" + nodeName + "/proxy/logs/?query=kubelet\""
+			for nodeName := range linuxNodeNames {
+				queryCommand := fmt.Sprintf("/api/v1/nodes/%s/proxy/logs/?query=kubelet", nodeName)
 				cmd := tk.KubectlCmd("get", "--raw", queryCommand)
 				result := runKubectlCommand(cmd)
 				assertContains("kubelet", result)
 			}
 		})
 
-		/*
-			Test if kubectl get --raw "/api/v1/nodes/<insert-node-name-here>/proxy/logs/?query=kubelet&pattern=container"
-			returns something or not!
-		*/
-
-		ginkgo.It("should return the logs filtered by pattern container", func(ctx context.Context) {
-			ginkgo.By("Starting the command")
-			tk := e2ekubectl.NewTestKubeconfig(framework.TestContext.CertDir, framework.TestContext.Host, framework.TestContext.KubeConfig, framework.TestContext.KubeContext, framework.TestContext.KubectlPath, ns)
-
-			for nodeName := range nodeNames {
-				queryCommand := "\"/api/v1/nodes/" + nodeName + "/proxy/logs/?query=kubelet&pattern=container\""
-				cmd := tk.KubectlCmd("get", "--raw", queryCommand)
-				result := runKubectlCommand(cmd)
-				assertContains("container", result)
-			}
-		})
-
 		/*
 			Test if kubectl get --raw "/api/v1/nodes/<insert-node-name-here>/proxy/logs/?query=kubelet&boot=0"
-			returns something or not!
+			returns kubelet logs from the current boot
 		*/
 
 		ginkgo.It("should return the kubelet logs for the current boot", func(ctx context.Context) {
 			ginkgo.By("Starting the command")
 			tk := e2ekubectl.NewTestKubeconfig(framework.TestContext.CertDir, framework.TestContext.Host, framework.TestContext.KubeConfig, framework.TestContext.KubeContext, framework.TestContext.KubectlPath, ns)
 
-			for nodeName := range nodeNames {
-				queryCommand := "\"/api/v1/nodes/" + nodeName + "/proxy/logs/?query=kubelet&boot=0\""
+			for nodeName := range linuxNodeNames {
+				queryCommand := fmt.Sprintf("/api/v1/nodes/%s/proxy/logs/?query=kubelet&boot=0", nodeName)
 				cmd := tk.KubectlCmd("get", "--raw", queryCommand)
 				result := runKubectlCommand(cmd)
 				assertContains("kubelet", result)
 			}
 		})
 
 		/*
-			Test if kubectl get --raw "/api/v1/nodes/<insert-node-name-here>/proxy/logs/?query=kubelet&boot=-1"
-			returns something or not!
+			Test if kubectl get --raw "/api/v1/nodes/<insert-node-name-here>/proxy/logs/?query=kubelet&tailLines=3"
+			returns the last three lines of the kubelet log
 		*/
 
-		ginkgo.It("should return the kubelet logs for the previous boot", func(ctx context.Context) {
+		ginkgo.It("should return the last three lines of the kubelet logs", func(ctx context.Context) {
+			ginkgo.Skip("FIXED")
```

Comment:
did you mean to leave the `ginkgo.SKip("FIXED")` in the PR?

Reply:
Sorry about that. Fixed.